apps/openmw/mwworld/store.cpp

```diff
@@ -1053,6 +1053,30 @@ namespace MWWorld
         }
     }
 
+    template<>
+    void Store<ESM::Static>::setUp()
+    {
+        // Load default marker definitions, if game files do not have them for some reason
+        std::pair<std::string, std::string> markers[] = {
+            std::make_pair("divinemarker", "marker_divine.nif"),
+            std::make_pair("doormarker", "marker_arrow.nif"),
+            std::make_pair("northmarker", "marker_north.nif"),
+            std::make_pair("templemarker", "marker_temple.nif"),
+            std::make_pair("travelmarker", "marker_travel.nif")
+        };
+
+        for (const std::pair<std::string, std::string> marker : markers)
+        {
+            const ESM::Static *ptr = search(marker.first);
+            if (ptr == 0)
```

Comment:
And while we are doing random code improvements, here is another one: These two lines could be written in a less verbose form:

if (!search (marker.first))